### ISSUE TITLE
Using ngx-gallery component in lazy module is not working in Angular 13

### ISSUE THREAD

**Issue by Vikas4790** [kolkov/ngx-gallery#88]
Using `ngx-gallery` works fine in a non lazy module, but throws `Error: BrowserModule has already been loaded. If you need access to common directives such as NgIf and NgFor from a lazy loaded module, import CommonModule instead.` when trying to access the component in a lazy module. 
Is anyone else facing the same issue?

It can be easily reproduced in a new angular13 project with `"@kolkov/ngx-gallery": "^2.0.0"` package version. 
Sample code of lazy loaded module - 
```
@NgModule({
  declarations: [Component2Component],
  imports: [
    NgxGalleryModule,
    RouterModule.forChild([{ path: "", component: Component2Component }]),
  ],
})
export class Module2Module {}
```

And then loading `Module2Module` lazily in app module like below throws the error when navigating to `http://localhost:4200/component2`- 
```
RouterModule.forRoot([
      ...
      {
        path: "component2",
        loadChildren: () =>
          import("./module2/module2.module").then((m) => m.Module2Module),
      },
    ]),
```

I feel this is related to the `BrowserAnimationsModule` getting imported alongside `NgxGalleryModule` when we add `NgxGalleryModule` to the imports: [...] of `Module2Module`. Since main app.module has already an import for `BrowserModule` so angular compiler throws up the error message stating that `BrowserModule has already been loaded`. Though I do not know how to fix this if this is the issue.

**Comment by randallmeeker** [comment]
This is the same in angular 9, rolling back version to version 1.2.3 fixed for me

**Comment by randallmeeker** [comment]
sure, was not saying this is not an issue, was just adding context. For me, it was 1.2.4 that broke my app.

**Comment by mnesser** [comment]
Same issue using lazy modules (v2.0.0). Works fine using v1.2.3.

**Comment by khvnkay** [comment]
any update ?

**Comment by andrejm7** [comment]
I had the same problem... I had to go back to the previous version...

**Comment by andrejm7** [comment]
@kolkov can u help? I went back to angular 12

**Comment by andrejm7** [comment]
I did the test here.. can you generate a new version without the IMPORT BrowserAnimationsModule ?

**Comment by andrejm7** [comment]
Please.. confirm pull request

**Comment by andrejm7** [comment]
with version 2.0.1 this problem has been fixed. thanks.

**Comment by Vikas4790** [comment]
This issue is resolved after the update 2.0.1. Marking it as closed. 
Thanks for the update @andrejm7 @kolkov .
